app/models/mixins/supports_feature_mixin.rb

```diff
@@ -94,11 +95,15 @@ module SupportsFeatureMixin
     :refresh_network_interfaces => 'Refresh Network Interfaces for a Host',
     :refresh_new_target         => 'Refresh non-existing record',
     :regions                    => 'Regions of a Provider',
+    :remove_all_snapshots       => 'Remove all snapshots',
     :remove_host                => 'Remove Host',
     :remove_interface           => 'Remove Interface',
+    :remove_snapshot            => 'Remove Snapshot',
+    :remove_snapshot_by_description  => 'Remove snapshot having a description',
```

Comment:
@durandom Thanks! I was thinking of adding a separate PR for adjusting the indentation. Should I?